cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -72,16 +75,22 @@ def test_vm_console(request, appliance, setup_provider, context, configure_webso
         myservice = MyService(appliance, service_name)
         vm_obj = myservice.launch_vm_console(catalog_item)
         vm_console = vm_obj.vm_console
+        if provider.one_of(OpenStackProvider):
+            public_net = provider.data['public_network']
+            provider.mgmt.assign_floating_ip(vm_obj.name, public_net)
         request.addfinalizer(vm_console.close_console_window)
         request.addfinalizer(appliance.server.logout)
+        ssh_who_command = ("who --count" if not provider.one_of(OpenStackProvider)
+                        else "who -aH")
```

Comment:
Please indent as described in our [Style Guide](http://cfme-tests.readthedocs.io/guides/dev_guide.html#code-style).

Reply:
Thanks for catching that.